### ISSUE TITLE
Generic text for Terms of Use page

### ISSUE THREAD

**Issue by stephaniesimms** [DMPRoadmap/roadmap#244]
https://uc3-roadmap-stg.cdlib.org/terms

Generic text for Roadmap Terms of Use in Google doc:
https://docs.google.com/document/d/11kxHalnf0IfEkUEkFpGfxaLlfD3thyUplhxhJWoxVvo/edit

**Comment by stephaniesimms** [comment]
@sjDCC Note to update the first paragraph 'Terms of use' - DCC is supported by ?

**Comment by sjDCC** [comment]
Hi @stephaniesimms What is this text intended for? Is it a generic dump to preload the roadmap code with in case of download by others? There are references to CDL and DCC at various points that I think should be ovewritten with generic content if so e.g. "It is hosted at the CDL" and "Further information about Cookies and how we use them is available on the main DCC website."

**Comment by sjDCC** [comment]
Suggested revisions

Terms of use
The Digital Curation Centre (DCC) and the California Digital Library (CDL) are consortia supported by the University of Edinburgh and the University of California, respectively. Our primary constituency is the research community. We provide services to the UK, US and international higher education sector.

DMPRoadmap
DMPRoadmap ('the tool', 'the system') is a tool developed by the DCC and CDL as a shared resource for the research community. 

Your personal details
In order to help identify and administer your account with DMPRoadmap, we need to store your email address. We may also use it to contact you to obtain feedback on your use of the tool, or to inform you of the latest developments or releases. The information may be transferred between the DCC and CDL partner institutions but only for legitimate DCC and CDL purposes. We will not sell, rent or trade any personal information you provide to us.

Privacy policy
The information you enter into this system can be seen by you, people you have chosen to share access with, and - solely for the purposes of maintaining the service - system administrators at DCC and CDL. We compile anonymized, automated and aggregated information from plans, but we will not directly access, make use of, or share your content with anyone else without your permission. Authorized officers of your home institution may access your plans for specific purposes - for example, to track compliance with funder/institutional requirements, to calculate storage requirements, or to assess demand for data management services across disciplines. 

Freedom of Information
[%ORGANISATION NAME] holds your plans on your behalf, but they are your property and responsibility. Any FOI applicants will be referred back to your home institution.

Passwords
Your password is stored in encrypted form and cannot be retrieved. If forgotten it has to be reset.

Cookies
Please note that DMPRoadmap uses Cookies. 

Third party APIs
Certain features on this website utilize third party services and APIs such as InCommon/Shibboleth or third party hosting of common JavaScript libraries or web fonts. Information used by an external service is governed by the privacy policy of that service. 

Revisions
This statement was last revised on July 22, 2016 and may be revised at any time. Use of the tool indicates that you understand and agree to these terms and conditions.

**Comment by sjDCC** [comment]
To summarise / clarify the above changes:

I've changed the DMPRoadmap - STAGE references to DMPRoadmap. Perhaps we want to replace them with a generic [%TOOL NAME] variable though so they're replaced by the service name more easily?

I removed the reference of where the tool is hosted to keep this generic but again this could be replaced by a variable string for ease of replacement.

Under the personal details and privacy policy paragraphs, do we want to remove the DCC and CDL references? I haven't for now.

I have removed the CDL privacy policy link under 'privacy policy' as that will surely differ per installation?

I have removed the DCC link under cookies as the policy will differ in each case I guess.

I have removed the Google Analytics info as this is surely bespoke so only relevant if host has installed?

I removed the CDL string under third-party apps

**Comment by stephaniesimms** [comment]
@sjDCC - We should also draft this is Google doc. I wrote the text above w/the consideration that we eventually want to use Roadmap-Stage as a sandbox. But after discussing w/ @briri we decided it should just be generic text for now (for external deployments) and can be amended in the future as needed. 

So all changes above are sound. Re first bullet, that is a generic [%TOOL NAME] string already so good there.
